velox/functions/sparksql/fuzzer/SparkAggregationFuzzerTest.cpp

```diff
@@ -88,6 +88,9 @@ int main(int argc, char** argv) {
       // coefficient. Meanwhile, DuckDB employs the sample kurtosis calculation
       // formula. The results from the two methods are completely different.
       "kurtosis",
+      // When all data in a group are null, Spark returns an empty array while
+      // DuckDB returns null.
+      "collect_list",
```

Comment:
@mbasmanova I think this function should not be compared with DuckDB. If the fuzzer generates a group where all the data is null, DuckDB's result will be null, while Spark will return an empty array.

Reply:
I agree. We need to change Fuzzer to verify results against Spark, not DuckDB: #9270